app/components/geoblacklight/header_component.html.erb

```diff
@@ -1,18 +1,14 @@
-<%= top_bar %>
-<% if not_landing_page? %>
-<%= search_bar %>
-<% else %>
-<div class='geobl-homepage-masthead jumbotron'>
+<header>
```

Comment:
Should we do this in Blacklight too? https://github.com/projectblacklight/blacklight/blob/main/app/components/blacklight/header_component.html.erb

Reply:
In Argo, we didn't put the search_bar in the `<header>`, it went in top_bar instead. I wonder if we can find consistency on that. https://github.com/sul-dlss/argo/blob/main/app/components/top_navbar_component.html.erb#L1

Reply:
I'm happy to adopt whatever Blacklight does 🤷🏻